dev/index.tsx

```diff
@@ -1,3 +1,38 @@
+/**
+Copyright [yyyy] [name of copyright owner]
```

Comment:
`2022 Immobiliare Labs` no?

Reply:
For now, I removed the license for every file.